lib/game.rb

```diff
@@ -41,46 +52,84 @@ def setup
     submarine_coordinates = gets.chomp.split(',')
 
     until_valid_placement(@player.submarine, submarine_coordinates)
-
-    puts @player.board.render(true)
   end
   
   def turn
+    both_boards_rendered
+    puts "Your turn:"
     puts "Enter coord to fire on"
```

Comment:
I had coord abbreviated because I was just quickly testing. We should probably update this to say coordinate.